VERSION

```diff
@@ -1 +1 @@
-v0.2.0
+v0.3.0-rc.1
```

Comment:
`VERSION` should not be manually updated. It gets updated with `make docs-release TAG=v${MAJOR_VERSION}.${MINOR_VERSION}` which should only be run for releases and not release candidates.

Reply:
VERSION needs to be updated so the version command reflects the proper version.